src/runtime/tests/runtime-test.ts

```diff
@@ -64,29 +63,29 @@ describe('Runtime', () => {
     const actual = await Runtime.getArcDescription(arc);
     assert.strictEqual(expected, actual);
   });
-  it('parses a Manifest', async () => {
-    const content = `
-    schema Greeting
-      value: Text
+  // it('parses a Manifest', async () => {
+  //   const content = `
+  //   schema Greeting
+  //     value: Text
 
-    particle Hello in 'hello.js'
-      text: writes Greeting {value}
+  //   particle Hello in 'hello.js'
+  //     text: writes Greeting {value}
 
-    recipe
-      handleA: create *
-      Hello
-        text: writes handleA`;
-    const expected = await Manifest.parse(content);
-    const actual = await Runtime.parseManifest(content);
-    assertManifestsEqual(actual, expected);
-  });
-  it('loads a Manifest', async () => {
-    const registry = {};
-    const loader = new Loader();
-    const expected = await Manifest.load('./src/runtime/tests/artifacts/test.manifest', loader, registry);
-    const actual = await Runtime.loadManifest('./src/runtime/tests/artifacts/test.manifest', loader, registry);
-    assertManifestsEqual(actual, expected);
-  });
+  //   recipe
+  //     handleA: create *
+  //     Hello
+  //       text: writes handleA`;
+  //   const expected = await Manifest.parse(content);
+  //   const actual = await Runtime.parseManifest(content);
+  //   assertManifestsEqual(actual, expected);
+  // });
+  // it('loads a Manifest', async () => {
+  //   const registry = {};
+  //   const loader = new Loader();
+  //   const expected = await Manifest.load('./src/runtime/tests/artifacts/test.manifest', loader, registry);
+  //   const actual = await Runtime.loadManifest('./src/runtime/tests/artifacts/test.manifest', loader, registry);
+  //   assertManifestsEqual(actual, expected);
+  // });
```

Comment:
Why are these commented out? If they're not needed, just delete. If there's some TODO required, please `it.skip` them with a comment.

Reply:
I just missed them in the sweep. Thanks.

Reply:
I decided to restore the tests, after all they were only ... wafer thin. Turned out to be more than I meant to bite off, but it all came out well in the end (apologies to Python, Monty).